### PR TITLE
fix broken link: http://kubernetes.io/docs/home/contribute/page-templ…

### PR DESCRIPTION
**Pull request by CriaHu**
…ates/

### CODE REVIEW

**Comment by tengqm** [comment]
@CriaHu Many thanks for fixing the 404 problems.
Please consider consolidate the many extra-small (XS) changes into batches when proposing PRs. For example, I see no reason why this one cannot be combined with #21679.

**Comment by CriaHu** [comment]
ok, I got it.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The [Kubernetes SIG Docs Discussion Group](https://groups.google.com/forum/#!for
 
 ### Style Guides and Templates
 
-Before submitting a pull request to create new content, please review the [Kubernetes.io style guide](http://kubernetes.io/docs/home/contribute/style-guide/) and follow the [instructions for using page templates](http://kubernetes.io/docs/home/contribute/page-templates/).
+Before submitting a pull request to create new content, please review the [Kubernetes.io style guide](http://kubernetes.io/docs/home/contribute/style-guide/) and follow the [instructions for using page templates](https://kubernetes.io/docs/contribute/style/page-content-types/).
```
Comment (kbhawkey) [inline CONTRIBUTING.md:21]: Hi @CriaHu . The doc pages are no longer using page templates. There are page content types.
What do you think about changing the link text to:
```
Before submitting a pull request to create new content, please review the [Kubernetes.io style guide](http://kubernetes.io/docs/home/contribute/style-guide/) and follow the [instructions about page content types](https://kubernetes.io/docs/contribute/style/page-content-types/).```

**Comment by kbhawkey** [comment]
Hi @CriaHu . Thanks for fixing the link to the site's information about page content types.
/lgtm

File: CONTRIBUTING.md (hunk 1 above)
Comment (kbhawkey) [reply CONTRIBUTING.md:21]: Let's go with the link update. Thank you!

**Pull request merged**